modules/nw-multus-advanced-annotations.adoc

```diff
@@ -0,0 +1,202 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/attaching-pod.adoc
+
+[id="nw-multus-advanced-annotations_{context}"]
+= Specifying pod-specific addressing and routing options
+
+When attaching a pod to an additional network, you may wish to specify further properties
+about that network in a particular pod. This allows you to change some aspects of routing, as well
+as specify static IP addresses and MAC addresses. In order to accomplish this, you may use the JSON formatted annotations.
+
+.Prerequisites
+
+* The Pod must be in the same namespace as the additional network.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
```

Comment:
We are in the process of updating our guidance on this. To make things easier, this can be "Instlal the OpenShift CLI (`oc`)."